src/components/ErrorBoundary/ErrorBoundary.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import * as logging from '../../Logging';
+import ErrorBoundary from './index';
+
+global.newrelic = {
+  addPageAction: jest.fn(),
+  noticeError: jest.fn(),
+};
+
+const welcomeError = new Error('Catch me if you can!');
+
+function Welcome() {
+  return <h1>Hi!</h1>;
+}
+
+describe('<ErrorBoundary />', () => {
+  it('renders childern correctly when no error', () => {
+    const tree = renderer
+      .create((
+        <ErrorBoundary>
+          <Welcome name="MA" />
+        </ErrorBoundary>
+      ))
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('logs the error', () => {
+    const logErrorMock = jest.fn();
+    logging.logError = logErrorMock;
```

Comment:
Unnecessary to create the `logErrorMock` variable since your assertion below does not assert no it; it asserts that logging.logError is called with the specific error, but not `logErrorMock`. You could just do:
```js
logging.logError = jest.fn();
```